src/api/index.js

```diff
@@ -1,4 +1,4 @@
-export const BASE_URL = 'https://classtranscribe-dev.ncsa.illinois.edu/api/'
+export const BASE_URL = 'https://classtranscribe.illinois.edu/api/'
```

Comment:
Just double checking, when we want to switch to using the dev server as the backend, we would have to manually change this link? Is there a way we can just call a flag when we run yarn start or something similar like that?

Reply:
Would it be appropriate to move BASE_URL to constants.js so that it's easier to manually change between production and test? Actually, why not import constants.js and do BASE_URL = '{FILE_SERVER_BASE_URL}/api/'

Reply:
Yep, good idea, I'll change that